scripts/hid_configurator/README.rst

```diff
@@ -151,6 +163,17 @@ Complete the following steps:
       pip3 install --user -r requirements.txt
       pip3 install --user .
 
+.. note::
+   Currently, the ``imgtool`` from PyPI does not support pure ED25519 signature (used by nRF54L-based devices that enable MCUboot hardware cryptography).
+   This may result in rejecting proper DFU images (``DFU image is invalid``).
+   ``imgtool`` supporting pure ED25519 signature can be installed from ``sdk-mcuboot`` repository (:file:`ncs/bootloader/mcuboot/scripts` directory of the |NCS|).
+   Run the following commands in the source directory to install ``imgtool`` and the required dependencies:
```

Comment:
We could add these texts as includes as these are repetition of line nos : 94-98. (This can be taken care in a separate PR as well).

.. include:: 
   :start-after
   :end-before: